launch.go

```diff
@@ -280,7 +288,7 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 
 	// Create meta space
 	log.Printf("Creating Meta Space in %v", metaSpace)
```

Comment:
This log will be always printed even if we use local mode and meta space is not created.

Reply:
I moved this log into `createMetaSpace` like any other logs about `Meta`